commands/group.go

```diff
@@ -194,7 +201,13 @@ func channelGroupListCmdF(c client.Client, cmd *cobra.Command, args []string) er
 		return errors.New("Unable to find channel '" + args[0] + "'")
 	}
 
-	groups, res := c.GetGroupsByChannel(channel.Id, 0, 9999)
+	groupOpts := model.GroupSearchOpts{
+		PageOpts: &model.PageOpts{
+			Page:    0,
+			PerPage: 10,
```

Comment:
Bad `PerPage` value:

```suggestion
			PerPage: 9999,
```